Makefile

```diff
@@ -42,21 +43,53 @@ setup:
 	read SHOOT; \
 	echo "enter cluster provider(gcp|aws|azure|vsphere|openstack|alicloud|metal|equinix-metal)"; \
 	read PROVIDER; \
-	./hack/local_setup.sh --seed $$SEED --shoot $$SHOOT --project $$PROJECT --provider $$PROVIDER
+	./hack/gardener_local_setup.sh --seed $$SEED --shoot $$SHOOT --project $$PROJECT --provider $$PROVIDER
 
-.PHONY: local-mcm-up
-local-mcm-up: setup
+.PHONY: gardener-local-mcm-up
+gardener-local-mcm-up: gardener-setup
 	$(MAKE) start;
 
-.PHONY: restore
-restore:
+.PHONY: gardener-restore
+gardener-restore:
 	@echo "enter project name"; \
 	read PROJECT; \
 	echo "enter shoot name"; \
```

Comment:
it doesn't ask for `SEED`

Reply:
the script does not need it. Where do you think seed should be used?

Reply:
ok you are targetting the control plane directly 
```
gardenctl target --garden "${virtual_garden_cluster}" --project "${PROJECT}" --shoot "${SHOOT}" --control-plane
```